### PR TITLE
Added dauricchio coords

### LINKED ISSUE #3061: Learing
Hello :wave:! Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by dauricchio**
@githubteacher, added my coords during 

> on demand training

closes #3061

### CODE REVIEW

**Comment by rookieGit** [comment]
@dauricchio looks good. but i'm not why the checks are failing. i'm getting the same fails.

File: _pins/dauricchio.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: dauricchio
```
Comment (crichID) [inline _pins/dauricchio.json]: @dauricchio the reason the tests are failing is that you have a space at the beginning of each line. If you click the **Files changed** tab, you will see a small pencil at the top right of the file. Click that and you will be back in edit mode! Please let me know if I can help!

**Comment by BrianHCombes** [comment]
Learned why the tests are failing here from crichID.
Nice!

**Comment by crichID** [comment]
@user I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3065/commits/1c8f5436b1e9c60ec3047d8dce8c1228e690da36. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**